src/Initiator.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace Deviantintegral\Har;
+
+use JMS\Serializer\Annotation as Serializer;
+
+/**
+ * @see https://developers.google.com/web/tools/chrome-devtools/network/reference#requests
```

Comment:
👏

Reply:
Thanks for providing the great base to start from!